lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -321,6 +321,10 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
   /// the result.
   lldb::TypeSP LookupClangType(llvm::StringRef name);
 
+  /// Returns a DependentGenericParamType with the desired depth and index.
+  CompilerType GetDependentGenericParamType(unsigned int depth,
```

Comment:
I think this should be a virtual function in TypeSystemSwift like `  CreateTupleType` and should use the same naming convention `CreateDependentGenericParamType`.